contracts/GenArt721CoreV3.sol

```diff
@@ -338,31 +344,57 @@ contract GenArt721CoreV3 is ERC721, Ownable, IGenArt721CoreContractV3 {
     }
 
     /**
-     * @notice Updates additional payee for project `_projectId` to be
-     * `_additionalPayee`, receiving `_additionalPayeePercentage` percent
-     * of artist mint and royalty revenues.
+     * @notice Updates additional payees and percentage splits for project
+     * `_projectId`.
+     * @param _projectId Project ID.
+     * @param _additionalPayeePrimarySales Address that may receive a
+     * percentage split of the artit's primary sales revenue.
+     * @param _additionalPayeePercentagePrimarySales Percent of artist's
+     * portion of primary sale revenue that will be split to address
+     * `_additionalPayeePrimarySales`.
+     * @param _additionalPayeeRoyalties Address that may receive a percentage
+     * split of the secondary sales royalties.
+     * @param _additionalPayeePercentageRoyalties Percent of artist's portion
+     * of secondary sale royalties that will be split to address
+     * `_additionalPayeeRoyalties`.
      */
-    function updateProjectAdditionalPayeeInfo(
+    function updateProjectAdditionalPayees(
         uint256 _projectId,
-        address payable _additionalPayee,
-        uint256 _additionalPayeePercentage
-    ) public onlyArtist(_projectId) {
-        require(_additionalPayeePercentage <= 100, "Max of 100%");
```

Comment:
Did we still want to double-check that primary sales & 2ndary sales %'s don't exceed 100 in this method? Maybe there's a separate check somewhere else and I missed it 😅

Reply:
nice catch! updated to check that the percentages are `<= 100` 🙏